metricbeat/mb/mb.go

```diff
@@ -206,24 +205,20 @@ func (b *BaseMetricSet) HostData() HostData {
 // the metricset fetches not only the predefined fields but add alls raw data under
 // the raw namespace to the event.
 type ModuleConfig struct {
-	Hosts      []string                `config:"hosts"`
-	Period     time.Duration           `config:"period"     validate:"positive"`
-	Timeout    time.Duration           `config:"timeout"    validate:"positive"`
-	Module     string                  `config:"module"     validate:"required"`
-	MetricSets []string                `config:"metricsets" validate:"required"`
-	Enabled    bool                    `config:"enabled"`
-	Filters    processors.PluginConfig `config:"filters"`
```

Comment:
We'll need documentation updates too for this since `filters` is gone? If so please drop a note into https://github.com/elastic/beats/issues/4540.